### ISSUE TITLE
Undefined method 'agree' when trying to destroy environment

### ISSUE THREAD

**Issue by ronny** [playup/dew#1]
```
#<DewCommand:0x00000101842d60 @invocation_path="dew", @context={}, @remaining_arguments=[], @subcommand_name="environments", @subcommand_arguments=["destroy", "some-prototype-86753"]>

/Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/lib/dew/controllers/environments_controller.rb:34:in `destroy': undefined method `agree' for #<EnvironmentsController:0x00000100910248> (NoMethodError)
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/lib/dew/commands/environments.rb:98:in `execute'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:63:in `run'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/subcommand/execution.rb:18:in `execute_subcommand'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:72:in `execute'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:63:in `run'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/subcommand/execution.rb:18:in `execute_subcommand'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:72:in `execute'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/lib/dew/base_command.rb:22:in `execute'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/bin/dew:31:in `execute'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:63:in `run'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/bin/dew:25:in `run'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/clamp-0.2.0/lib/clamp/command.rb:119:in `run'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/gems/dew-0.1.0/bin/dew:87:in `<top (required)>'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/bin/dew:19:in `load'
        /Users/ronny/.rvm/gems/ruby-1.9.2-p180@dew/bin/dew:19:in `<main>'
```

**Comment by ronny** [comment]
A workaround is to specify -f to skip confirmation.

**Comment by playupchris** [comment]
Fixed in [v0.1.1](http://github.com/playup/dew/tree/v0.1.1) 10f7ffc6a69b6211eae2a84834ee2c44370f3ffe
